react/components/Pagination/PageIndicator.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function PageIndicator({ currentItemFrom, itemTo, textOf, totalItems }) {
+  return (
+    <div className="c-muted-2 t-small">
+      {currentItemFrom}
+      {' - '}
+      {itemTo} {textOf} {totalItems}
+    </div>
```

Comment:
you can use this:
```jsx
    <div className="c-muted-2 t-small">
        {`${currentItemFrom} - ${itemTo} ${textOf} ${totalItems}`}
    </div>
```

Reply:
Partially done. One can aggregate the `${currentItemFrom} - ${itemTo}` part, but not the others, because in reality `textOf` can be a `<FormattedMessage />`, and implementing your suggestion exactly can render a string like `1 - 10 [object Object] 300` because the template string calls `.toString()` for its parameters.

Reply:
but `textOf` can never be `<FormattedMessage />` since your PropType expects it to be a `string`.

Reply:
That's true, but further inspection reveals that `Pagination/index.js` expects `PropTypes.node.isRequired` for `textOf`. If I change this, current behaviour can change for some of our dependants. Which means... we have a choice:
1. Risk causing display bugs for some apps that are using `<FormattedMessage />` and aggregate all props in the same template message as expected strings.
2. Change `textOf` prop to `node` in `PageIndicator.js` and keep things as they are.